### PR TITLE
Mandrel: Add 20.1.0.4, 20.3.1.2 and 21.0.0.0

### PR DESCRIPTION
**Pull request by zakkak**

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/java/MandrelMigrations.scala [hunk 1]
```diff
@@ -60,4 +60,58 @@ class MandrelMigrations {
       )
     ).validate()
       .insert()
+
+  @ChangeSet(
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/java/MandrelMigrations.scala:64]: You can do something like this

https://github.com/sdkman/sdkman-db-migrations/blob/master/src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala#L218-L241

**Review by eddumelendez — CHANGES_REQUESTED** [review]

File: src/main/scala/io/sdkman/changelogs/java/MandrelMigrations.scala (hunk 1 above)
Comment (zakkak) [reply src/main/scala/io/sdkman/changelogs/java/MandrelMigrations.scala:64]: Thanks for the pointer @eddumelendez

**Comment by eddumelendez** [comment]
@zakkak thanks for updating the PR! in order to pass the checks run this command `./gradlew scalafmt` please

**Comment by zakkak** [comment]
> @zakkak thanks for updating the PR! in order to pass the checks run this command `./gradlew scalafmt` please

Hi @eddumelendez,

Are you sure this is a formatting issue? Inspecting the [logs](https://github.com/sdkman/sdkman-db-migrations/pull/463/checks?check_run_id=1878213506#step:6:697) I see:
```
[main] ERROR io.sdkman.changelogs.package$$anon$1 - URL https://download.java.net/java/early_access/panama/3/openjdk-16-panama+3-385_linux-x64_bin.tar.gz responded with text/html content-type detected

[main] INFO com.github.mongobee.Mongobee - Mongobee is releasing process lock.
Error: Exception in thread "main" com.github.mongobee.exception.MongobeeException: Invalid url: https://download.java.net/java/early_access/panama/3/openjdk-16-panama+3-385_linux-x64_bin.tar.gz
	at com.github.mongobee.Mongobee.executeMigration(Mongobee.java:200)
	at com.github.mongobee.Mongobee.execute(Mongobee.java:156)
	at io.sdkman.DbMigration$.delayedEndpoint$io$sdkman$DbMigration$1(DbMigration.scala:10)
	at io.sdkman.DbMigration$delayedInit$body.apply(DbMigration.scala:5)
	at scala.Function0.apply$mcV$sp(Function0.scala:39)
	at scala.Function0.apply$mcV$sp$(Function0.scala:39)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)
	at scala.App.$anonfun$main$1$adapted(App.scala:80)
	at scala.collection.immutable.List.foreach(List.scala:431)
	at scala.App.main(App.scala:80)
	at scala.App.main$(App.scala:78)
	at io.sdkman.DbMigration$.main(DbMigration.scala:5)
	at io.sdkman.DbMigration.main(DbMigration.scala)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.github.mongobee.Mongobee.executeChangeSetMethod(Mongobee.java:235)
	at com.github.mongobee.Mongobee.executeMigration(Mongobee.java:181)
	... 12 more
Caused by: com.github.mongobee.exception.MongobeeChangeSetException: Invalid url: https://download.java.net/java/early_access/panama/3/openjdk-16-panama+3-385_linux-x64_bin.tar.gz
	at io.sdkman.changelogs.package$$anon$1.validUrl(package.scala:44)
	at io.sdkman.changelogs.package$$anon$1.validUrl(package.scala:28)
	at io.sdkman.changelogs.package$$anon$2.$anonfun$validUrl$1(package.scala:55)
	at io.sdkman.changelogs.package$$anon$2.$anonfun$validUrl$1$adapted(package.scala:55)
	at scala.collection.immutable.List.foreach(List.scala:431)
	at io.sdkman.changelogs.package$$anon$2.validUrl(package.scala:55)
	at io.sdkman.changelogs.package$$anon$2.validUrl(package.scala:51)
	at io.sdkman.changelogs.package$ValidationOps.validate(package.scala:91)
	at io.sdkman.changelogs.java.OpenJdkMigrations.migrate095(OpenJdkMigrations.scala:184)
	... 18 more
```

**Comment by eddumelendez** [comment]
You are right, sorry

**Comment by eddumelendez** [comment]
Depends on https://github.com/sdkman/sdkman-db-migrations/pull/467

**Comment by eddumelendez** [comment]
@zakkak can you please update your branch with master?

**Comment by marc0der** [comment]
@zakkak if you don't mind, please rebase against master, not merge master.

**Review by eddumelendez — CHANGES_REQUESTED** [review]
for windows the package extension is zip. I would suggest to use one migration per version.

**Comment by zakkak** [comment]
> for windows the package extension is zip. I would suggest to use one migration per version.

Hmm fixed the issue before seeing your message. Do you still want me to use one migration per version?

**Review by eddumelendez — APPROVED** [review]

**Pull request merged**